bindings/py/tests/sparse_matrix_test.py

```diff
@@ -162,6 +162,70 @@ def test_setAllNonZeros(self):
         error('setAllNonZeros, out of order')
 
 
+  def test_nNonZerosPerRow_allRows(self):
+    for name, denseMatrix, expected in (
```

Comment:
Fair enough. I use this strategy elsewhere in this file. I think it's useful for encouraging us to have a suite of inputs and outputs for each method. But you're right that it's not really benefiting this test, especially since testing this method is so easy.